device-types/Dell/PowerEdge-R760xd2.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Dell
+model: PowerEdge R760xd2
+slug: dell-poweredge-r760xd2
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+airflow: front-to-rear
+weight: 46.3
+weight_unit: kg
+comments: |
+  [PowerEdge R760xd2 spec sheet](https://www.delltechnologies.com/asset/fr-fr/products/servers/technical-support/poweredge-r760xd2-spec-sheet.pdf)
+  [PowerEdge R760xd2 tech guide](https://www.delltechnologies.com/asset/en-us/products/servers/technical-support/poweredge-r760xd2-technical-guide.pdf)
+rear-ports:
+  - name: Rear USB 2
+    type: other
+  - name: Rear USB 3
+    type: other
+  - name: iDrac
+    type: other
+front-ports:
+  - name: Front USB 2
+    type: other
+    rear_port: Rear USB 2
+  - name: Front USB micro-AB
+    type: other
+    rear_port: iDrac
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
```

Comment:
These PSUs are modular. Thus, they should be defined as `module-bays` for the device, and the associated PSUs should be added in the `module-types` folder